whole-tale/ngx-dashboard
# feat: convert login page to about page

## Problem
The login page previously contained our acknowledgments and a link to the TOS. We need a new place to make these available in the UI.

## Approach
Convert old `/login` view into new `/about` view

<img width="1441" alt="Screen Shot 2022-09-02 at 10 17 14 AM" src="https://user-images.githubusercontent.com/1413653/188181428-4be3797f-45d6-49b9-a949-ba28214e0f5e.png">


## How to Test
1. Checkout this branch locally, rebuild the dashboard
2. Navigate to the WholeTale Dashboard
    * In the navbar you should see a (?) icon
3. Click on the (?) icon
    * You should be brought to the new About page
    * You should see University and grant acknowledgments here
    * You should see a link to the Terms of Service
    * You should **not** see the old "Sign In" buttons in the center
